admin/tests/unit/adapters/schooling-registration_test.js

```diff
@@ -16,7 +16,7 @@ module('Unit | Adapter | schooling-registration', function (hooks) {
     test('it performs the request to dissociate user from student', async function (assert) {
       // given
       const schoolingRegistration = { id: 12345 };
```

Comment:
Suggestion : Pourquoi pas en profiter pour changer le schooling ici en organization learner 
```suggestion
      const OrganizationLearner = { id: 12345 };
```